src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -77,6 +77,7 @@ param(
   [parameter(Mandatory=$true, Position=1)][string] $destination,
   [parameter(Mandatory=$false, Position=2)][string] $specificFolder,
   [parameter(Mandatory=$false, Position=3)][string] $packageName,
+  [parameter(Mandatory=$false, Position=4)][switch] $recursivecall,

```

Comment:
Please add the documentation for this up in the docs above. Please observe the way it is done currently and just add to it.